internal/cmd/tui.go

```diff
@@ -339,6 +341,41 @@ func (m tuiModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			}
 
 			return m, tea.Quit
+
+		case "ctrl+w", "w":
```

Comment:
Why use the `w` keybind? That usually corresponds to saving (writing) in vim or nano. I think forward slash "`/`" would be a more fitting choice (that is regex search in vim). `f` for "find" is also not a bad choice.

Reply:
Perfect I will go with `f` and `crl+f`

Reply:
BTW in nano W means Where is
![image](https://github.com/stateful/runme/assets/2709271/7f3bc56a-e29f-4e0c-a884-ae88a0283388)